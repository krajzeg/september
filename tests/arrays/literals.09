print("Should print 1, 2, 3.")
for (element) in ([1,2,3]) {
	print(element)
}

print("Should not print anything.")
for (element) in ([]) {
	print(element)
}
