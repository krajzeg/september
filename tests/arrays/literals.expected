Should print 1, 2, 3.
1
2
3
Should not print anything.
